### DISCUSSION TITLE
Compile against OpenLDAP-2.5.4

### DISCUSSION THREAD

**Discussion by 3eka** [curl/curl#7003] [Q&A] [answered]
Hi all,

OpenLDAP has released 2.5.4 version and curl compiling fails... Any chance to get it adopted soon?
Regards.

**Comment by bagder** [comment, marked as answer]
I think filing this as an issue and providing all the details would be a good start...

**Reply by bagder** [reply]
I filed one now: https://github.com/curl/curl/issues/7004
